proto/commonfate/access/v1alpha1/access.proto

```diff
@@ -316,6 +317,8 @@ message GrantState {
 message Validation {
   // If true, a reason is required for access.
   bool has_reason = 1;
+  // List of regex patterns that the reason must conform with
+  repeated RegexValidation regex = 2;
```

Comment:
Maybe something slightly more descriptive here. Although it is hard to convey what the regex is supposed to do / relate to based on its name
```suggestion
  repeated RegexValidation reason_regex = 2;
```